docs/api/repository-signatures-resource.md

```diff
@@ -63,15 +61,19 @@ The following request fetches the repository signatures index.
 
 The repository signature index is a JSON document that contains an object with the following properties:
 
-Name                | Type             | Required
-------------------- | ---------------- | --------
-allRepositorySigned | boolean          | yes
-signingCertificates | array of objects | yes
+Name                | Type             | Required | Notes
+------------------- | ---------------- | -------- | -----
+allRepositorySigned | boolean          | yes      | Must be `false` on 4.7.0 resource
+signingCertificates | array of objects | yes      | 
 
 The `allRepositorySigned` boolean is set to false if the package source has some packages that have no repository
 signature. If the boolean is set to true, all packages available on the source must have a repository
 signature produced by one of the signing certificates mentioned in `signingCertificates`.
 
+> [!Warning]
+> The `allRepositorySigned` boolean should be false on the 4.7.0 resource. NuGet v4.7 and v4.8 clients cannot install packages
+> from sources that have `allRepositorySigned` set to true.
```

Comment:
See https://github.com/NuGet/NuGet.Client/pull/2471 and https://github.com/NuGet/NuGet.Client/pull/2468. I didn't list these issues in the documentation as I didn't think they are particularly helpful

Reply:
I would use "must" instead of "should"